src/index.html

```diff
@@ -1,17 +1,97 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8">
+    <meta charset="UTF-8" >
     <meta
       name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" >
     <title>HTML Form</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./style.css" >
   </head>
+
   <body>
-    <h1>HTML Form</h1>
+    <form action ="https://mate-academy-form-lesson.herokuapp.com/create-application" method="POST">
+      <fieldset class="fieldset" >
+        <legend>Personal information</legend>
+        <label class="info">Surname:
+          <input type="text" name="surname"
+          autocomplete="off"
+          >
```

Comment:
```suggestion
          <input 
            type="text" 
            name="surname"
            autocomplete="off"
          >
```
all properties should start on a new line (checklist p3)
please fix it everywhere